source/MaterialXRenderGlsl/GLTextureHandler.cpp

```diff
@@ -295,4 +289,45 @@ int GLTextureHandler::getNextAvailableTextureLocation()
     return -1;
 }
 
+
+int GLTextureHandler::mapAddressModeToGL(ImageSamplingProperties::AddressMode addressModeEnum)
```

Comment:
Minor code shuffle to move these to the bottom and keep maintenance logic together.